pkg/patch/patch.go

```diff
@@ -386,3 +404,29 @@ func dockerLoad(ctx context.Context, pipeR io.Reader) error {
 
 	return cmd.Run()
 }
+
+// e.g. "docker.io/library/nginx:1.21.6-patched".
+func getRepoNameWithDigest(ctx context.Context, patchedImageName string) (string, error) {
+	cmd := execCommandContext(ctx, "docker", "image", "inspect",
```

Comment:
can we do this without docker cli?

Reply:
I’m wondering if we could make use of the BuildKit exporter response, since it should include the digest of the patched image. The challenge is that this information only becomes available once we’ve finished generating the VEX report, because everything happens within the same build command. I’d really appreciate any tips or guidance on how to make that work!

Reply:
Do we need to generate the vex from within the build?